mailer.js

```diff
@@ -130,7 +136,8 @@ module.exports = function (log) {
         link: link,
         oneClickLink: oneClickLink,
         supportUrl: this.supportUrl,
-        supportLinkAttributes: this._supportLinkAttributes()
+        supportLinkAttributes: this._supportLinkAttributes(),
+        passwordChangeLinkAttributes: this._initiatePasswordChange()
```

Comment:
Should `_initiatePasswordChange` be renamed to `_passwordChangeLinkAttributes` to stay consistent?